centos-quarkus-maven/Dockerfile

```diff
@@ -5,34 +5,37 @@ FROM centos:7
 
 ARG GRAAL_VERSION=1.0.0-rc14
 ARG MAVEN_VERSION=3.6.0
-
-ENV GRAAL_CE_URL=https://github.com/oracle/graal/releases/download/vm-${GRAAL_VERSION}/graalvm-ce-${GRAAL_VERSION}-linux-amd64.tar.gz
+ARG USER_HOME_DIR="/root"
```

Comment:
Why using root?

Reply:
no big reason just aligned with default user i.e. root, can change it

Reply:
changed this to `/quarkus`

Reply:
I see the home directory need to be set while creating the user so defaulting to `/root` which is the $HOME for `root` which is used while running the build. We can update it to other users when we no longer need root to run buildah.